tools/wt_verify/wt_verify.py

```diff
@@ -614,10 +626,15 @@ def main():
         print(pretty_output)
 
     if "dump_blocks" in command:
-        imgs = show_block_distribution_broken_barh(args.filename, parsed_data)
-        imgs += show_block_distribution_hist(args.filename, parsed_data)
-        imgs += show_free_block_distribution(args.filename, parsed_data)
-        serve(imgs)
+        try:
+            imgs = show_block_distribution_broken_barh(args.filename, parsed_data)
+            imgs += show_block_distribution_hist(args.filename, parsed_data)
+            imgs += show_free_block_distribution(args.filename, parsed_data)
+            serve(imgs)
+        except Exception as e:
+            print(str(e), file=sys.stderr)
+            print(traceback.format_exc(), file=sys.stderr)
```

Comment:
Same thing here - could we just "let it fail" without the `try`/`except` machinery?